.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,23 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: ""
+labels: 'enhancement'
+assignees: ''
+
+---
+
+## **Feature request**
+Note: We are happy to provide support to the entire community, however our [members and sponsors](https://mobilitydata.org/members/) have their issues prioritized as a benefit of their membership. MobilityData staff is aware of all issues opened, therefore there is no need for you to tag us directly. 
```

Comment:
```suggestion
<!-- Note: We are happy to provide support to the entire community, however our [members and sponsors](https://mobilitydata.org/members/) have their issues prioritized as a benefit of their membership. MobilityData staff is aware of all issues opened, therefore there is no need for you to tag us directly.  -->
```

Reply:
adding those symbols will prevent for displaying in the issue itself and will reduce the noice

Reply:
Thanks for pointing that!